apps/mull-api/src/app/user/user.resolver.ts

```diff
@@ -29,8 +29,9 @@ export class UserResolver {
     return this.userService.getAllUsers();
   }
 
+  @UseGuards(AuthGuard)
```

Comment:
Thanks for making these changes. At first it made sense to use @authticateduser, but since these queries are needed for other users too its best to simply have the id as a variable :/